content/en/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm.md

```diff
@@ -30,7 +30,7 @@ etcd cluster of three members that can be used by kubeadm during cluster creatio
   document assumes these default ports. However, they are configurable through
   the kubeadm config file.
 - Each host must have systemd and a bash compatible shell installed.
-- Each host must [have a container runtime, kubelet, and kubeadm installed](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/).
+- Each host must have a container runtime, kubelet, and [kubeadm installed](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/).
```

Comment:
my vote goes to:
```suggestion
- Each host must have a [container runtime, kubelet, and kubeadm installed](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/).
```

because the linked page talks about all of them.

Reply:
My opinion also same as @neolit123 here.